node/engine/chainservice/eth_chainservice.go

```diff
@@ -132,6 +146,31 @@ func newEthChainService(chain ethChain, na *NitroAdjudicator.NitroAdjudicator,
 	return &ecs, nil
 }
 
+func (ecs *EthChainService) checkForMissedEvents(startBlock uint64) error {
+	ecs.logger.Info("checking for missed chainservice events", "startBlock", startBlock)
+	query := ethereum.FilterQuery{
```

Comment:
Totally optional but now that we're constructing a `FilterQuery` in two places we could consider adding a helper function to create a `FilterQuery`

Reply:
The two instances of `FilterQuery` that we currently use are different enough that I don't think a helper function accomplishes much. I propose we add a helper function in the future if we have more instances of `FilterQuery` and the helper function saves us lines of code